### DISCUSSION TITLE
Availability of Raspberry Pi boards

### DISCUSSION THREAD

**Discussion by aviljoenn** [bareboat-necessities/lysmarine_gen#303] [General]
What's the take on BBN and the availability issues around the Raspberry Pi boards?
Is there thinking around making a supported image available for a different SBC other than Raspberry Pi?

**Comment by mgrouch** [comment]
Which SBC do you have in mind?

You can buy raspberry pi if you shop for it as pre-built with enclosure. There are some edatec CM4 based industrial Pis with interfaces more suitable for boat in stock on digikey.

**Reply by aviljoenn** [reply]
I was thinking possibly the Odroid N2 as a higher end device.
It seems that HA Home Assistant also leans toward the Odroid N2.
Other smaller option is the Odroid C4
Both of those are very good boards.
